test/parallel/test-runner-output.mjs

```diff
@@ -14,12 +14,13 @@ function replaceSpecDuration(str) {
   return str
     .replaceAll(/\(0(\r?\n)ms\)/g, '(ZEROms)')
     .replaceAll(/[0-9.]+ms/g, '*ms')
-    .replaceAll(/duration_ms [0-9.]+/g, 'duration_ms *');
+    .replaceAll(/duration_ms [0-9.]+/g, 'duration_ms *')
+    .replace(new RegExp(`(\\u001b\\[\\d+m)\\(${process.cwd()}/?(\\u001b\\[\\d+m)(.*)(\\u001b\\[\\d+m)\\)`, 'g'), '$3');
 }
 const defaultTransform = snapshot
   .transform(snapshot.replaceWindowsLineEndings, snapshot.replaceStackTrace, replaceTestDuration);
 const specTransform = snapshot
-  .transform(snapshot.replaceWindowsLineEndings, snapshot.replaceStackTrace, replaceSpecDuration);
+  .transform(replaceSpecDuration, snapshot.replaceWindowsLineEndings, snapshot.replaceStackTrace);
```

Comment:
Why did we have to change the ordering of this transform? Would it be helpful to document this with a comment?